benchmarks/maxtext_xpk_runner.py

```diff
@@ -333,6 +336,43 @@ def build_user_command(
   return command
 
 
+def _get_pathways_specific_flags(wl_config: WorkloadConfig):
+  pw_config = wl_config.pathways_config
+  if pw_config is None:
+    return ''
+
+  remote_python_sidecar_image_flag = (
+      f' --remote-python-sidecar-image={pw_config.remote_python_sidecar_image}'
+      if pw_config.remote_python_sidecar_image is not None
+      else ''
+  )
+  server_image_flag = (
+      f' --server-image={pw_config.server_image}'
+      if pw_config.server_image is not None
+      else ''
+  )
+  proxy_image_flag = (
+      f' --proxy-server-image={pw_config.proxy_image}'
```

Comment:
Nit - Could we keep XPK flags and runner flags be names similar, so that there are lesser human errors while using the benchmark runner?

Reply:
Fixed.